app/models/setting.rb

```diff
@@ -68,6 +68,14 @@ def self.[](name)
     end
   end
 
+  def self.has_value?(name)
```

Comment:
nil is not a valid value to be "cached" in Rails.cache. it treats it as a cache miss. thus, caching again :cry: